.github/workflows/lint-autoformat.yml

```diff
@@ -19,14 +19,17 @@ jobs:
         with:
           submodules: true
           fetch-depth: 0
-      - name: Setup miniconda
-        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
-        with:
-          python-version: "3.10"
+      - name: Install python
```

Comment:
Could we just use system python here or does it need to be higher than a certain version, i.e. 3.10?

Reply:
ok yeah seems fine